### PR TITLE
new record

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: PRS2001-3000/PRS2907Brocchi.xml [hunk 1]
```diff
@@ -44,14 +45,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                  Brocchi da Imola
-               </persName>
-               <occupation> dipl. </occupation>
+            <person sex="1" sameAs="wd:Q16639164">
+               <persName xml:lang="it" xml:id="n1">Giovanni Battista Brocchi da Imola</persName>
+               <persName xml:lang="la" xml:id="n2">Johannes Baptista Brocchus</persName>
+               <occupation type="political">Diplomat</occupation>
+               <floruit notBefore="1481" notAfter="1493">Nobleman and diplomat who accompanied an Ethiopian delegation from <placeName ref="LOC3957Jerusa">Jerusalem</placeName>
+                  to <placeName ref="LOC5374Rome">Rome</placeName> in <date>1481–1482</date>. Often conflated with his contemporary, the similarily named messenger
+                  <persName ref="PRS14337Baptista"/>.
```
Comment (eu-genia) [inline PRS2001-3000/PRS2907Brocchi.xml:54]: I think if we start with bibliography we should start with Cerulli (Etiopi in Palestina or at least https://www.treccani.it/enciclopedia/giovanni-battista-brocchi_res-fe73653f-87e8-11dc-8e9d-0016357eee51_(Dizionario-Biografico)/)

I am not so convinced that Cerulli was confused

Comment (eu-genia) [reply PRS2001-3000/PRS2907Brocchi.xml:54]: I would follow Cerulli and stay with one Giovanni Battista Brocchi da Imola.

Comment (eu-genia) [reply PRS2001-3000/PRS2907Brocchi.xml:54]: @abausi when you have a second you can also share your thoughts )

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: PRS2001-3000/PRS2907Brocchi.xml (hunk 1 above)
Comment (thea-m) [reply PRS2001-3000/PRS2907Brocchi.xml:54]: I find Verena's arguments against this view convincing :)

File: PRS2001-3000/PRS2907Brocchi.xml [hunk 2]
```diff
@@ -44,14 +45,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                   Giovanni Battista
-                  Brocchi da Imola
-               </persName>
-               <occupation> dipl. </occupation>
+            <person sex="1" sameAs="wd:Q16639164">
+               <persName xml:lang="it" xml:id="n1">Giovanni Battista Brocchi da Imola</persName>
+               <persName xml:lang="la" xml:id="n2">Johannes Baptista Brocchus</persName>
+               <occupation type="political">Diplomat</occupation>
+               <floruit notBefore="1481" notAfter="1493">Nobleman and diplomat who accompanied an Ethiopian delegation from <placeName ref="LOC3957Jerusa">Jerusalem</placeName>
+                  to <placeName ref="LOC5374Rome">Rome</placeName> in <date>1481–1482</date>. Often conflated with his contemporary, the similarily named messenger
```
Comment (eu-genia) [inline PRS2001-3000/PRS2907Brocchi.xml:53]: This is not so sure.

File: PRS2001-3000/PRS2907Brocchi.xml [hunk 3]
```diff
@@ -44,14 +45,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                  <persName ref="PRS14337Baptista"/> (see the literature overview provided by Krebs).
+                  Contrary to that person, Giovanni Battista Brocchi da Imola most likely never visited Ethiopia.
+               </floruit>
             </person>
+            <listRelation>
+               <relation name="syriaca:different-from" active="PRS2907Brocchi" passive="PRS14337Baptista"/>
```
Comment (eu-genia) [inline PRS2001-3000/PRS2907Brocchi.xml:59]: on the contrary, if you insist on creating a second record, we need to show in the relation that they may rather be the same person.

File: new/PRS14337Baptista.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            <change who="DR" when="2024-02-14">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="en" xml:id="n1">Baptista of Imola</persName>
+                    <floruit notBefore="1480" notAfter="1484">Messenger who carried letters between <placeName ref="LOC3957Jerusa">Jerusalem</placeName> and 
+                        <placeName ref="LOC3010Ethiop">Ethiopia</placeName> in the period 1480–1484, not to be confused with the similarily named nobleman
```
Comment (eu-genia) [inline new/PRS14337Baptista.xml:50]: quite probably the same

File: new/PRS14337Baptista.xml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="en" xml:id="n1">Baptista of Imola</persName>
+                    <floruit notBefore="1480" notAfter="1484">Messenger who carried letters between <placeName ref="LOC3957Jerusa">Jerusalem</placeName> and 
+                        <placeName ref="LOC3010Ethiop">Ethiopia</placeName> in the period 1480–1484, not to be confused with the similarily named nobleman 
+                        <persName ref="PRS2907Brocchi"/> who was active in the same period.</floruit>
+                </person>
+                <listRelation>
+                    <relation name="syriaca:different-from" active="PRS14337Baptista" passive="PRS2907Brocchi"/>
```
Comment (eu-genia) [inline new/PRS14337Baptista.xml:54]: as above

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: PRS2001-3000/PRS2907Brocchi.xml (hunk 2 above)
Comment (thea-m) [reply PRS2001-3000/PRS2907Brocchi.xml:53]: The identity as one person stated by Conti Rossini etc. is the traditional opinion based mostly upon Lefevre's research. I see no general reason to give precedence to this view over the reevaluation of sources carried out by Verena, unless there are concrete arguments.

**Comment by abausi** [comment]
We will mention Verena Krebs' hypothesis, obviously, but since hers is one view among others (also modern authors have not favourably commented on that), it is more functional to remain with one, where all information, also on the debate, will be immediately available.

**Pull request merged**